utils/apiCalls/authorization.ts

```diff
@@ -0,0 +1,37 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { FIREBASE_API_KEY } from 'react-native-dotenv';
+
+const firebaseAuth = axios.create({
+  baseURL: 'https://identitytoolkit.googleapis.com/v1/',
+  headers: { 'Content-Type': 'application/json' },
+});
+
+const config: AxiosRequestConfig = {
+  params: {
+    key: FIREBASE_API_KEY,
+  },
+};
+
+export const loginUser = (email, password) => {
+  const data = {
+    email,
+    password,
+    returnSecureToken: true,
+  };
+
+  return firebaseAuth
+    .post(`/accounts:signInWithPassword`, data, config)
+    .then(response => response.data);
+};
+
+export const signupUser = (email: string, password: string, phone?: string) => {
```

Comment:
`phone` parameter is not used here. Can we include it in the request?
If it's not needed for now, I would remove it as it gives ESLint error :smiley:

Reply:
I will include it :+1: